mlir/test/Dialect/Mesh/ops.mlir

```diff
@@ -164,6 +164,16 @@ func.func @mesh_shard_shape() {
   return
 }
 
+// CHECK-LABEL: func @mesh_get_sharding
+// CHECK-SAME: %[[ARG:.*]]: tensor<4x8xf32>
+func.func @mesh_get_sharding(%arg0 : tensor<4x8xf32>) -> !mesh.sharding {
+  // CHECK-NEXT: %[[S:.*]] = mesh.sharding @mesh1 split_axes = {{\[\[}}], [0]] : !mesh.sharding
+  %s = mesh.sharding @mesh1 split_axes = [[], [0]] : !mesh.sharding
```

Comment:
`%s` is unused and unrelated, why do you need it in this test?

Reply:
Probably a lef-over from a different approach. Removed.
We need a shardingOp when lowering, though.